static/app/views/replays/types.tsx

```diff
@@ -1,6 +1,7 @@
 export type Replay = {
```

Comment:
More TS shenanigans.. .
`projectID` isn't fetched, and `replayId` seems to be empty / not used any of the time.

Reply:
Oh good catch I'll get those removed.

Reply:
Alright removed those.